src/js/jsx/sections/export/LayerExports.jsx

```diff
@@ -0,0 +1,297 @@
+                this.props.document, this.props.layer, this.props.index, scaleNum);
+        },
+
+        /**
+         * Update this asset's suffix
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateSuffix: function (event, suffix) {
+            this.getFlux().actions.export.updateLayerAssetSuffix(
+                this.props.document, this.props.layer, this.props.index, suffix);
+        },
+
+        /**
+         * Update this asset's format
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateFormat: function (format) {
+            var formatLower = format && format.toLowerCase();
+
+            this.getFlux().actions.export.updateLayerAssetFormat(
+                this.props.document, this.props.layer, this.props.index, formatLower);
+        },
+
+        render: function () {
+            var layer = this.props.layer,
+                exportAsset = this.props.exportAsset,
+                scale = exportAsset.scale || 1,
+                scaleOption = scaleOptions.has(scale.toString()) ?
+                    scaleOptions.get(scale.toString()) : scaleOptions.get("1"),
+                scaleListID = "layerExportAsset-scale" + layer.id + "-" + this.props.index,
+                formatListID = "layerExportAsset-format-" + layer.id + "-" + this.props.index;
+
+            return (
+                <div className="formline">
+                    <Datalist
+                        list={scaleListID}
+                        className="dialog-export-scale"
+                        options={scaleOptions.toList()}
+                        value={scaleOption.title}
+                        onChange={this._handleUpdateScale}
+                        live={false}
+                        size="column-3" />
+                    <Gutter />
+                    <TextInput
+                        value={exportAsset.suffix}
+                        singleClick={true}
+                        editable={true}
+                        onChange={this._handleUpdateSuffix}
+                        size="column-6" />
+                    <Gutter />
+                    <Datalist
+                        list={formatListID}
+                        className="dialog-export-format"
+                        options={formatOptions.toList()}
+                        value={exportAsset.format.toUpperCase()}
+                        onChange={this._handleUpdateFormat}
+                        live={false}
+                        size="column-4" />
+                    <Gutter />
+                    <Button
+                        className="button-plus" // a bit of a hack
+                        title="remove asset configuration"
+                        onClick={this._handleDeleteClick}>
+                        <SVGIcon
+                            viewbox="0 0 12 14"
+                            CSSID="libraries-delete" />
```

Comment:
Confirmed with seth it should be generic, so I'm changing the name and both references (export and library)